client/src/redux/sagas/systems.sagas.js

```diff
@@ -26,6 +27,10 @@ function* pushSystemKeys(action) {
         props: {}
       }
     });
+    if (modalRefs.FileSelector) {
+      yield call(modalRefs.FileSelector.props.toggle);
+      yield put({ type: 'CLEAR_REFS' });
+    }
```

Comment:
Do we only care about the `FileSelector` ref? If so, should we not store every ref, and only store the latest FileSelector ref?

Reply:
Currently yes. This structure leaves room for other refs to be added. Would it be better to do away with the `refs` object in the data files state for now?

Reply:
I would say to maybe either only store the ref we need, if there's no immediate need for any other ref, or to do away with storing refs and use state instead. What are your thoughts?

Reply:
Added a follow up ticket: https://jira.tacc.utexas.edu/browse/FP-1035

Reply:
cool! 👍  adding a note in the ticket just to check/consider (or possibly ignore) this [comment](https://github.com/TACC/Core-Portal/pull/219#pullrequestreview-513988672) whenever that ticket is worked on.